yt_dlp/extractor/funker530.py

```diff
@@ -0,0 +1,75 @@
+    # TODO: `python test/test_download.py TestDownload.test_Funker530` only runs the first test?
+    _TESTS = [{
+        'url': 'https://funker530.com/video/azov-patrol-caught-in-open-under-automatic-grenade-launcher-fire/',
+        'md5': '085f50fea27523a388bbc22e123e09c8',
+        'info_dict': {
+            'id': 'v2qbmu4',
+            'ext': 'mp4',
+            'title': 'Azov Patrol Caught In Open Under Automatic Grenade Launcher Fire',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'uploader': 'Funker530',
+            'channel': 'Funker530',
+            'channel_url': 'https://rumble.com/c/c-1199543',
+            'width': 1280,
+            'height': 720,
+            'fps': 25,
+            'duration': 27,
+            'upload_date': '20230608',
+            'timestamp': 1686241321,
+            'live_status': 'not_live',
+            'description': 'md5:bea2e1f458095414e04b5ac189c2f980',
+        }
+    }, {
+        'url': 'https://funker530.com/video/my-friends-joined-the-russians-civdiv/',
+        'md5': '',
+        'info_dict': {
+            'id': '',
+            'ext': 'mp4',
+            'title': 'My “Friends” Joined the Russians - CivDiv',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'uploader': '',
+            'width': '',
+            'height': '',
+            'fps': '',
+            'duration': '',
+            'upload_date': '',
+            'description': 'md5:',
+        }
+    }]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+        rumble = RumbleEmbedIE(downloader=self._downloader)
+        youtube = YoutubeIE(downloader=self._downloader)
+        found_rumble_videos = rumble._extract_embed_urls(url, webpage)
+        found_yt_videos = list(youtube._extract_embed_urls(url, webpage))  # TODO: is converting it to a list OK if there's only one video?
+        if len(list(found_yt_videos)):
+            embedded_video = youtube.extract(found_yt_videos[0])  # TODO: do I need to do all found videos or is [0] ok?
+        elif len(found_rumble_videos):
+            embedded_video = rumble.extract(found_rumble_videos[0])  # TODO: see above
+            desc_regex = re.compile(r'(?s)<div class="row video-desc-paragraph">.*?<p>(.*?)(About the Author|<\/div>\n?<\/div>\n?<\/div>)')
+            # _html_search_regex is not cleaning <style> tags so we'll do it ourselves.
+            # description = self._html_search_regex(desc_regex, webpage, 'description', fatal=False)
+            description = re.search(desc_regex, webpage).group(1)
+            if description:
+                description = clean_html(re.sub(r'<style>(.|\s)*?<\/style>', '', description))
+
+            embedded_video['description'] = description
+        else:
+            # TODO: is this correct if nothing is found?
+            return
+        return embedded_video
```

Comment:
If we're going to return a url result, then we want `'_type': 'url_transparent'` in the return so that we're not extracting the description for no reason.

We can use some `utils` like `get_element_by_class` and `strip_or_none` to reduce the regex load.
And use `_search_regex` with `default=None` to make it non-fatal
```suggestion

        return {
            **info,
            '_type': 'url_transparent',
            'description': strip_or_none(self._search_regex(
                r'(?s)(.+)About the Author', clean_html(get_element_by_class('video-desc-paragraph', webpage)),
                'description', default=None))
        }
```